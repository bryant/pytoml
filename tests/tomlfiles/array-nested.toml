twoarrays     = [ [ 1, 2 ], ["a", "b", "c"] ] # this is ok
