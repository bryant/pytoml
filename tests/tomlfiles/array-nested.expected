{'twoarrays': [[1, 2], ['a', 'b', 'c']]}